### DISCUSSION TITLE
Welcome to OpenCNCPilot Discussions!

### DISCUSSION THREAD

**Discussion by martin2250** [martin2250/OpenCNCPilot#168] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by deHarro** [comment]
Nice idea! 👍 
Harald

**Comment by silicon-systems** [comment]
Thanks Martin.

Is this to restructure previous discussions or just the usual solutions to OpenCNCPilot **how to.**

Hanspeter

**Comment by martin2250** [comment]
Hello Hanspeter,

I want to restructure discussions, meaning mostly separating actual issues from discussions / project showcases / help wanted. With this new space, it's also easier to manage multiple discussions compared to having them all in one very long issue.

What do you mean with "the usual solutions to OpenCNCPilot how to"?

Cheers

**Reply by silicon-systems** [reply]
> What do you mean with "the usual solutions to OpenCNCPilot how to"?
> 
> Cheers

Hi Martin,

It's the **project showcases** & **usual help** to OpenCNCPilot, is what I meant.

Kind regards.
